static/js/vue-cdr-access/src/mixins/routeUtils.js

```diff
@@ -109,6 +109,16 @@ export default {
          */
         nonDuplicateNavigationError(error) {
             return (error.name !== 'NavigationDuplicated' && !/^avoided\s+redundant\s+navigation/i.test(error.message));
+        },
+
+        /**
+         * Vite really, really wants all images to be in the project being built and referenced via imports.
+         * This doesn't make sense for some of our images. The project won't build unless an import or url is given.
+         * So just return the image url for images that are external to the Vue project.
+         * @param image
+         */
+        nonVueStaticImageUrl(image) {
```

Comment:
I'm only seeing this being used in one place, do you think it'd make sense to copy `ajax-loader-lg.gif` over into the project to make vue happy?

Reply:
I'm inclined to leave it. We'd have to move over an additional copy of the image during deploys and/or adjust the the image reference in the compiled js.